README.md

```diff
@@ -75,12 +75,12 @@ process, which runs as UID 200.
 
 * There is an environment variable that can used to pass JVM arguments to the startup script
 
-  * `INSTALL4J_ADD_VM_PARAMS`, passed to the Install4J startup script. Defaults to `-Xms1200m -Xmx1200m -XX:MaxDirectMemorySize=2g`.
+  * `INSTALL4J_ADD_VM_PARAMS`, passed to the Install4J startup script. Defaults to `-Xms1200m -Xmx1200m -XX:MaxDirectMemorySize=2g -Djava.util.prefs.userRoot=${NEXUS_DATA}/javaprefs`.
```

Comment:
It might be worth adding a practical note about why this is exposed, something akin to "this will prevent your license from being wiped out when you restart Nexus Repository using our Docker image".

Reply:
@DarthHater Feel free to massage the readme to what seems appropriate.

Reply:
In: 0d5f9dd